CMakeLists.txt

```diff
@@ -1027,6 +1027,15 @@ hpx_option(
   ADVANCED
 )
 
+hpx_option(
+  HPX_WITH_NANOBENCH
+  BOOL
+  "Use Nanobench for performance tests. Nanobench will be fetched using FetchContent (defualt: OFF)"
```

Comment:
typo :)

Reply:
Fixed in https://github.com/STEllAR-GROUP/hpx/pull/6448/commits/c35e4e4f3ca2332c6f08323c9e3504175bede47d